packages/create-sitecore-jss/watch.json.example

```diff
@@ -12,6 +12,9 @@
     "language": "da-DK",
     "fetchWith": "GraphQL",
     "prerender": "SSG",
-    "hostName": "sitecore-jss-nextjs.dev.local"
+    "hostName": "sitecore-jss-nextjs.dev.local",
+    // if you're developing packages i.e install dependencies before initializing an app.
+    // Set devTemplates to false or else your yarn.lock will get restored.
+    "devTemplates": true
```

Comment:
I think we should tie this config name directly to the action it's controlling. "restoreLockfile"?